CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing to r7insight_docker
+
+:+1::tada: Thanks for taking the time to contribute! :tada::+1:
+
+## Workflow
+
+- Fork repository in GitHub
+- Clone your repository fork
+- Work on functionality
+- `make build` for building the Docker image
+- `make test` for testing the built image
+- `make clean` for removing the built docker image and it's tags
+
+Once you have finished implementing the desired functionality, you should open a Pull Request on GitHub  
+Once it is approved for merging, you should bump the versioning (this repository uses [SemVer](https://semver.org/)) running by whichover one of the following that makes sense:
+- `make bump-major`
+- `make bump-minor`
+- `make bump-patch`
+
+Push the bump commit into the GitHub PR.
+At this stage, the Rapid7 team should approve, merge and deploy the new package.
+
+## Deployment/Publishing
```

Comment:
We should probably add a note that only R7 staff will be able to publish

Reply:
I was thinking this will be a general how to publish/deploy for 3rd parties who care to deploy their own